test/renate/data/test_data_module.py

```diff
@@ -156,3 +132,48 @@ def test_tiny_imagenet_data_module(tmpdir):
     assert len(test_data) == num_te
     assert isinstance(test_data, Dataset)
     assert train_data[0][0].size() == test_data[0][0].size() == (3, 64, 64)
+
+
+@pytest.mark.parametrize(
+    "dataset_name,input_column,target_column",
+    [
+        ("rotten_tomatoes", "text", "label"),
+    ],
+)
+@pytest.mark.parametrize(
+    "tokenizer", [transformers.DistilBertTokenizer.from_pretrained("distilbert-base-uncased")]
+)
+@pytest.mark.parametrize(
+    "tokenizer_kwargs",
+    [
+        None,
+        dict(padding="max_length", max_length=128, truncation=True),
```

Comment:
When passing None, the default configuration will be used and the default is the same configuration you specify here. It would be good to run the test with two different configurations.

Reply:
Good point.